.github/workflows/tests.yml

```diff
@@ -15,13 +15,13 @@ jobs:
       matrix:
         os:
           - ubuntu-20.04
-        ruby-version: [2.6, 2.7, 3.0]
+        ruby-version: ['2.6', '2.7', '3.0', '3.1']
```

Comment:
Not quoting this causes the version to be parsed as a float, which can have unintended results.

Reply:
I've added Ruby 3.1 here to ensure that we can run on the latest version.